Releases/LatestRelease/FindSimilarIncidents.md

```diff
@@ -0,0 +1,3 @@
+Fix bugs:
```

Comment:
please change to be as documented here https://github.com/demisto/internal-content/tree/master/documentation/change-log